ipn/localapi/localapi.go

```diff
@@ -153,6 +153,12 @@ type Handler struct {
 	// (operator user).
 	PermitWrite bool
 
+	// PermitLocalAdmin is whether the client may write machine-wide state to
```

Comment:
These "permit" names are kinda confusing in retrospect.

This might read more nicely (if inconsistent) as "CallerIsLocalAdmin" or something.